x-pack/plugins/security_solution/public/assistant/get_comments/index.tsx

```diff
@@ -41,7 +59,7 @@ export const getComments = ({
       ) : (
         <EuiAvatar name="machine" size="l" color="subdued" iconType="logoSecurity" />
       ),
-      timestamp: i18n.AT(message.timestamp),
+      timestamp: i18n.AT(showAnonymizedValues ? message.timestamp : transformedMessage.timestamp),
```

Comment:
This is the timestamp of the message itself, I don't think this needs anonymized?